src/pages/ImportData.js

```diff
@@ -392,7 +393,28 @@ const ImportData = () => {
     };
   };
 
-  const importTdrExport = (namespace, name) => {
+  const loadWdsUrl = useCallback((workspaceId) => {
+    return Ajax()
+      .Apps.listAppsV2(workspaceId)
+      .then(resolveWdsUrl)
+      .then((url) => {
+        return url;
+      });
```

Comment:
nit: do you need the second `.then` here? Could this be just:
```javascript
return Ajax().Apps.listAppsV2(workspaceId).then(resolveWdsUrl);
```
of course if you are able to refactor the methods in `Data.js` to be reusable this comment is moot